docs/collections/documents/tutorials/using-realtime-updates.md

```diff
@@ -67,73 +67,72 @@ Let's assume your
 <TabItem value="py" label="Python">
 
 ```py
-  from c8 import C8Client
-  import threading
-  import pprint
-  import time
-
-  # Variables - URLs
-  global_url = "gdn.paas.macrometa.io"
-
-
-  # Variables - DB
-  email = "nemo@nautilus.com"
-  password = "xxxxx"
-  geo_fabric = "_system"
-  collection_name = "ddos"
-
-  # Variables - Data
-  data = [
-      {"ip": "10.1.1.1", "action": "block", "rule": "blacklistA"},
-      {"ip": "20.1.1.2", "action": "block", "rule": "blacklistA"},
-      {"ip": "30.1.1.3", "action": "block", "rule": "blacklistB"},
-      {"ip": "40.1.1.4", "action": "block", "rule": "blacklistA"},
-      {"ip": "50.1.1.5", "action": "block", "rule": "blacklistB"},
-    ]
+import time
+import threading
+import pprint
+from c8 import C8Client
+
+# Variables - URLs
+GLOBAL_URL = "gdn.paas.macrometa.io"
```

Comment:
Why are you making the variables uppercase? I thought lowercase was the accepted standard.

Reply:
According to [PEP](https://peps.python.org/pep-0008/#constants), constants should be written in uppercase. The support team has been making changes for the pyC8 audit using as a guide the previous link to keep an internal standard. There is another [file](https://github.com/Macrometacorp/docs/blob/master/docs/streams/tutorials/using-realtime-updates.md) within the docs, which has the same content as this. Maybe part of the concern is that we have a #Variables comment in the python file, but they are constants since they do not change their value over time for this example.

Reply:
Cool. I didn't realize variables needed to change value in order to be variables.  Learn something new every day. :)

Would they not be variables because the constant would change for different users? Like my email is different than your email, so our variable/constant would have a different value.

Reply:
I understand your point, and I think it can even get more complex. Maybe the word I should use is mutable and immutable objects in python. But in another context, we were also suggesting these changes because of the use of a Linter, in this case, Pylint. It led us to keep almost the same structure for all the files.

Reply:
Hey, I like consistency. Thanks for explaining it. :)

Reply:
You have an approval, so feel free to merge this once all tests pass.